ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -619,34 +619,22 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: baremetal-lab-upi
-- as: gcp-ipi-oidc-rt-fips-amd-f14
-  cron: 25 18 1,17 * *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      EXTRACT_MANIFEST_INCLUDED: "true"
-      FIPS_ENABLED: "true"
-      RT_ENABLED: "true"
-    test:
-    - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
 - as: gcp-ipi-mini-perm-custom-type-f28
   cron: 43 8 27 * *
   steps:
     cluster_profile: gcp-qe
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-minimal-permission
-- as: gcp-ipi-oidc-rt-fips-regen-cert-f28
-  cron: 9 16 13 * *
+- as: gcp-ipi-oidc-rt-fips-amd-f14
```

Comment:
I am using f28 for all upgrade jobs, do you think f14 is a little often?

Reply:
Done

Reply:
I am seeing this job is being edit in your PR, so I will revert my change in this PR.